.azure-pipelines/stages.yml

```diff
@@ -70,6 +70,7 @@ stages:
   - template: stage/linux.yml
     parameters:
       cacheTestResults: ${{ parameters.cacheTestResults }}
+      bazelConfigRBE: --config=remote-ci --config=rbe-google --jobs=200
```

Comment:
Why hardcoding number of jobs?

Reply:
currently it gets this only in postsubmit - i think what we want is for it to always get this - to  make it faster (higher prio) than the checks

Reply:
yeah - this is a separate issue - checks tend to swamp RBE because there is a lot of them, but nothing depends on them, so this is more to prioritize the build jobs over checks - not really related to pre/post